atm.js

```diff
@@ -1,3 +1,185 @@
+  $("#savings .withdraw").on("click", function(e){
+      e.preventDefault();
+      //sets balance equal to whatever is in the checking account and makes it a number
+      balance = $("#savings .balance").html();
+      nBalance = parseFloat(balance.split("$")[1]);
+
+      //sets userAmount to whatever is in the input and makes it a  number
+      userAmount = $("#savings .input").val();
+      userAmountN = parseFloat(userAmount);
+
+      withdrawSavings();
+
+      $("#savings .input").val("");
+
+    makeSavingsRed();
+    ifSWithdrawAtStart();
+    overDrawSavings();
+
+
+    $("#savings .input").val("");
+  })
+
+// savings functions
+
+function makeSavingsRed () {
+  if (totalBalance === 0) {
+    $("#savings .balance").addClass("zero");
+  } else if (totalBalance > 0) {
+    $("#savings .balance").removeClass("zero");
+  }
+}
+
+function overDrawSavings () {
+  if (totalBalance < 0) {
+    overDrawAmount = userAmountN - nBalance;
+    addToChecking();
+    $("#savings .balance").html("$" + "0");
+  }
+}
+
+function ifSWithdrawAtStart() {
+  if (userAmount === "") {
+    $("#savings .balance").html(balance);
+    }
+}
+
+
+function sNotRed() {
+  if (totalBalance > 0) {
+    $("#savings .balance").removeClass("zero");
+  }
+}
+
+function withdrawSavings() {
+  totalBalance = nBalance - userAmountN;
+  $("#savings .balance").html("$" + totalBalance.toFixed(2));
+}
+
+function depositSavings() {
+  totalBalance = nBalance + userAmountN;
+  $("#savings .balance").html("$" + totalBalance.toFixed(2));
+}
+
+// Identical Checking Functions
+
+function makeCheckingRed () {
+  if (totalBalance === 0) {
+    $("#checking .balance").addClass("zero");
+  }
+}
+
+function overDrawChecking () {
+   if (totalBalance < 0) {
+    overDrawAmount = userAmountN - nBalance;
+    addToSavings();
+    $("#checking .balance").html("$" + "0");
+}
+}
+
+function ifCWithdrawAtStart() {
+  if (userAmount === "") {
+    $("#checking .balance").html(balance);
+    }
+}
+
+
+function cNotRed() {
+  if (totalBalance > 0) {
+    $("#checking .balance").removeClass("zero");
+  }
+}
+
+function withdrawChecking() {
+  totalBalance = nBalance - userAmountN;
+  $("#checking .balance").html("$" + totalBalance.toFixed(2));
+}
+
+function depositChecking() {
+  totalBalance = nBalance + userAmountN;
+  $("#checking .balance").html("$" + totalBalance.toFixed(2));
+}
+
+function addToChecking() {
+    var checkingTotal = $("#checking .balance").html();
+    var checkingTotalNumber = parseFloat(checkingTotal.split("$")[1]);
+    var newCheckingTotal = checkingTotalNumber - overDrawAmount;
+    checkingNewTotal = $("#checking .balance").html("$" + newCheckingTotal.toFixed(2));
+  }
+
+function addToSavings() {
+  var savingsTotal = $("#savings .balance").html();
+  var savingsTotalNumber = parseFloat(savingsTotal.split("$")[1]);
+  var newSavingsTotal = savingsTotalNumber - overDrawAmount;
+  savingsNewTotal = $("#savings .balance").html("$" + newSavingsTotal.toFixed(2));
+}
+
```

Comment:
Make sure to account for the scenario where you overdraft from savings to try and withdraw from checking! Right now, it's going negative